### ISSUE TITLE
Copyright Picture text box Styles don't act the same between Light and None

### ISSUE THREAD

**Issue by davidc86** [sillsdev/ptx2pdf#896]
First off it looks like the default font size factor for the "None|x-credit" style is 1.00 rather than the 0.42, which both Dark|x-credit and Light|x-credit use. (Is this a mistake, or is there a reason?)

If I reduce the size of the Font Size of "None|x-credit" to 0.42 (to match the "Lightlx-credit" box specs), I don't see any change in the (c) text for the "None|x-credit" boxes.

**Comment by davidg-sil** [comment]
There shouldn't be anything magic going on in any code... but both issues suggest to me that the settings from the user-interface are getting overridden. Might it be that you've got an entry in `ptxprint-mods.sty` or some other style sheet?

**Comment by davidc86** [comment]
None of our PTXprint projects don't use any external .sty files. Just using the Styles Tab.

BTW, while probably not relevant, I'll give this anyway, the ptxprint-mods.tex file has this in it:

% This is the .tex file specific for the SLU project used by PTXprint.
% Saved Configuration name: SelaruNewPublication
\TOCthreetabfalse
\clubpenalty=0
\widowpenalty=0 
\NoteCallerSpace=0.0em
% \noteRag=0pt plus 2fill
% The following sets copyright text box inside height/width around font (default {1pt})
% \def\FigCreditPadding{0pt}

So, again (with v2.3.45) it is still the case that the three "|x-credit styles" are not identical in definition (except for font color and background color):

Dark and Light are identical for all settings but color. Specifically relevant is that the Font Size Factor is set to 0.42

Whereas the None|x-credit style's Font Size Factor is set to 1.00  -- and changing this to match the Dark and Light styles marks this setting with BOLD, meaning it deviates from the "default". But for Dark and Light, a factor of 0.42 is the default.

I don't know who or what decides what is "default" for the various styles, but clearly the 3 styles don't agree!

And last time I checked the "none" didn't pay any attention to the Style tab anyway.

**Comment by davidg-sil** [comment]
Oh, how embarrassing:

```

\Marker Dark|x-credit
\Name Dark|x-credit - Auxiliary - Figure credit box styling
\Description Styles the credit box for a figure
\OccursUnder None
\FontName Charis SIL
\FontSize 5
\Color xFFFFFE
\Background x000000

\Marker Light|x-credit
\Name Light|x-credit - Auxiliary - Figure credit box styling
\Description Styles the credit box for a figure
\OccursUnder None
\FontName Charis SIL
\FontSize 5
\Background xFFFFFE

\Marker None|x-credit
\Name None|x-credit - Auxiliary - Figure credit box styling
\Description Styles the credit box for a figure
\OccursUnder None
\FontName Charis SIL
\Color x000000
\Background -
```

So, you're correct in that `None|x-credit` doesn't have the `FontSize` setting (5/12 = roughly, 0.42).  
But on the other hand,  it ought to be falling through to the the value for x-credit (which has the `FontSize` setting). I'm thus very confused why it doesn't fall through to that.  I'll have to check to see what the code thinks it is up to.

**Comment by davidg-sil** [comment]
Right... I think what we have here is not a bug, but insufficient documentation, and ideally some contextual change to the user-interface would be nice.
`Light/Dark/None|x-credit` styles (conceptually) are add-ons to the x-credit style, much like character styles are add-ons to whatever is set in the paragraph style.
If a value is not set in `Light/Dark/None/Custom|x-credit` style, then the value from x-credit style should take effect.
This is why, when you set the size of `None|x-credit` from **nothing set** to 0.42, you didn't see any change.  The UI displaying 1.0 (11pt)  when there's nothing set  is ... _really unhelpful_, because that assumes the fallback value is `\Fontsize 12`, from `\p`'s settings, but it's not for these styles.

It is more appropriate, therefore, if the `Light/Dark/None/Custom|x-credit` styling use 'scale relative to parent' and new / unmodified projects will have that set once I've done the push.  The parent, in that case is `x-credit`, and I've updated the description to say that.

![image](https://github.com/sillsdev/ptx2pdf/assets/58848232/747586c4-e8e0-40a4-8ea9-54db892b41b8)

**Comment by davidg-sil** [comment]
I've just noticed, however that the check-box 'scale to parent' isn't ticked, even though the value is displayed as a fontscale. Grr, and it's still showing 11pt, when it's not. I guess that bit must count as a bug.

**Comment by davidg-sil** [comment]
The user-interface bug (displaying the wrong modifier, etc) is still there. It might be a Martin-type bug, but hopefully not...
* Scale relative to parent tickbox not loaded at start-up 
* Calculation is incorrect. When using a complex style which sets a fontscale, it's not scaled relative to `\p`, it's relative to the base parameter the complex style is based upon - in this case `x-credit`.
